utils/sync-dependencies.ts

```diff
@@ -114,14 +128,21 @@ function main(argv: string[]) {
 
     // Dependency ranges used by the consumer of Perseus (like khan/frontend)
     const clientVersionRanges = filterUnusableTargetVersions(
-        JSON.parse(fs.readFileSync(clientPackageJson).toString()).dependencies,
+        clientPackageJson.dependencies,
         packageNamesInRepo,
     );
 
+    function getClientVersionRange(pkgName: string) {
```

Comment:
I don't know how to describe this, but there's something weird about the naming here. There's `clientVersionRanges` and `getClientVersionRange`, but we don't use `getClientVersionRange` to set `clientVersionRanges`; rather we use `clientVersionRanges` as an argument for `getClientVersionRange`.

I think maybe `clientVersionRanges` is no longer descriptive of what it is? It's part of the story but not the whole story.